server/utils/monitoring.ts

```diff
@@ -10,7 +10,6 @@ export function init() {
 	Sentry.init({
 		dsn: process.env.SENTRY_DSN,
 		environment: process.env.NODE_ENV,
-		tracesSampleRate: process.env.NODE_ENV === 'production' ? 1 : 0,
```

Comment:
I think it would make sense to move this prod check to the `tracesSampler` function